### PR TITLE
MAUI-892140 Add Legend Toggle and Stacked Area Marker Support content to UG Documentation

### PR DESCRIPTION
**Pull request by subash-sf**
This PR updates the User Guide (UG) to include new content regarding the following features:

•	**Legend Toggle Support**: Added instructions and examples on how to use the legend toggle feature in the following charts, allowing users to easily show or hide legends as needed.
                     o	SfPyramidChart
                     o	SfFunnelChart
•	**Marker Support**: Included details on how to utilize marker support for the following charts, enabling users to add and customize markers for better data visualization.
                     o	Stacked Area Chart
                     o	Stacked Area 100 Chart

These additions aim to enhance the documentation and provide clearer guidance on leveraging these features effectively.

Changes Made:
•	Updated UG sections related to chart customization.
•	Added examples and usage scenarios for legend toggle and marker features.

### CODE REVIEW

File: MAUI/Cartesian-Charts/StackedArea.md [hunk 1]
```diff
@@ -97,4 +97,104 @@ this.Content = chart;
...
+    ...
+</chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfCartesianChart chart = new SfCartesianChart();
+...
+ChartMarkerSettings chartMarker= new ChartMarkerSettings();
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/StackedArea.md:179]: set chartMarker instance to chart marker setting 

Reference -https://help.syncfusion.com/maui/cartesian-charts/line#marker-customization
image

File: MAUI/Cartesian-Charts/StackedArea100.md [hunk 2]
```diff
@@ -100,4 +100,104 @@ this.Content = chart;
...
+    ...
+</chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfCartesianChart chart = new SfCartesianChart();
+...
+ChartMarkerSettings chartMarker= new ChartMarkerSettings();
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/StackedArea100.md:182]: set chartMarker instance to chart marker setting

Reference -https://help.syncfusion.com/maui/cartesian-charts/line#marker-customization
image

**Review by SaiyathAliFathima — CHANGES_REQUESTED** [review]

File: MAUI/Cartesian-Charts/StackedArea.md (hunk 1 above)
Comment (subash-sf) [reply MAUI/Cartesian-Charts/StackedArea.md:179]: - Set the chartMarker instance to MarkerSettings for the necessary charts.
- Modified the mentioned changes under migration section for all the five charts.

File: MAUI/Cartesian-Charts/StackedArea100.md (hunk 2 above)
Comment (subash-sf) [reply MAUI/Cartesian-Charts/StackedArea100.md:182]: Set the chartMarker instance to MarkerSettings for the necessary charts.
Modified the mentioned changes under migration section for all the five charts.

**Review by SaiyathAliFathima — APPROVED** [review]
File changes are fine

**Review by Saravanan-Madhesh — APPROVED** [review]
Approved, the style fixes checks in staging.

**Pull request merged**